cypress/integration/version/unscheduled_patch/configure_patch.ts

```diff
@@ -516,14 +516,18 @@ describe("Configure Patch Page", () => {
       cy.server();
       cy.visit(`/patch/${unactivatedPatchId}`);
     });
-    it("Clicking `Schedule` button schedules patch and redirects to patch page", () => {
+    it("Clicking 'Schedule' button schedules patch and redirects to patch page", () => {
       const val = "hello world";
       cy.dataCy(`patch-name-input`).as("patchNameInput").clear().type(val);
       cy.dataCy("task-checkbox").first().check({ force: true });
-      // TODO: Replace this with cy.intercept() once we upgrade to > Cypress 6.0.0
-      cy.route2("/graphql/query", (req) => {
+      cy.intercept("/graphql/query", (req) => {
         req.reply((res) => {
-          res.body = mockedSuccessfulConfigureResponse;
+          res.body = {
+            data: {
+              schedulePatch: { versionFull: { id: unactivatedPatchId } },
+            },
+            errors: null,
```

Comment:
nit. use a variable for mockedSuccessfulConfigureResponse since we do the same thing for the error response.